cmd/utils/flags.go

```diff
@@ -2084,6 +2084,7 @@ func MakeChain(ctx *cli.Context, stack *node.Node) (chain *core.BlockChain, chai
 
 	// TODO(rjl493456442) disable snapshot generation/wiping if the chain is read only.
 	// Disable transaction indexing/unindexing by default.
+	// PSP - Check for config.ParallelEVM.Enable here?
```

Comment:
Are those reminders for next iterations?

Reply:
Nice catch! The todo is not required anymore. Will remove this comment.

Reply:
[Removed](https://github.com/maticnetwork/bor/pull/840/commits/adce0e823527da26822a4076e1a977c6bc6e6d00).